src/Promitor.Scraper.Tests.Unit/Serialization/v1/V1IntegrationTests.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using AutoMapper;
+using Microsoft.Azure.Management.Monitor.Fluent.Models;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Promitor.Core.Scraping.Configuration.Model.Metrics.ResourceTypes;
+using Promitor.Core.Scraping.Configuration.Serialization;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Core;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Mapping;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.ResourceTypes;
+using Xunit;
+using ResourceType = Promitor.Core.Scraping.Configuration.Model.ResourceType;
+
+namespace Promitor.Scraper.Tests.Unit.Serialization.v1
+{
+    [Category("Unit")]
+    public class V1IntegrationTests
```

Comment:
Can we call this unit test or serialization test? I prefer to keep integration test for running live infrastructure tests

Reply:
Sure - makes sense.